qa/tasks/cephfs/test_mirroring.py

```diff
@@ -585,7 +585,7 @@ def test_cephfs_mirror_stats(self):
         self.assertGreater(second["counters"]["snaps_synced"], first["counters"]["snaps_synced"])
         self.assertGreater(second["counters"]["last_synced_start"], first["counters"]["last_synced_start"])
         self.assertGreater(second["counters"]["last_synced_end"], second["counters"]["last_synced_start"])
-        self.assertGreater(second["counters"]["last_synced_duration"], 0)
+        self.assertGreaterEqual(second["counters"]["last_synced_duration"], 0)
```

Comment:
We've changed the code in PeerReplayer, so that sync_duration could have `0` seconds. So this^ metric check should be `>=0` even for bigger data set.

Reply:
That's fine. What I meant was if we create a large enough data set in test, then it would likely take more than a second for sync and thereby the check here will verify that the stat is indeed working.

Reply:
Fixed